testfixtures_test.go

```diff
@@ -100,6 +100,12 @@ func testLoader(t *testing.T, dialect, connStr, schemaFilePath string, additiona
 		if err := l.Load(); err != nil {
 			t.Errorf("cannot load fixtures: %v", err)
 		}
+
+		// Call load again to test against a database with existing data.
```

Comment:
I added a second call to `Load()` here because the UseAlterConstraint() behavior with DELETE CASCADEs would pass on the first run with an empty database. However on the second run with existing data it would fail. This is because where the DELETE CASCADES come into effect depending on the order of insertion.